### PR TITLE
[Install] Add Swift 6.0 nightly Windows 10 arm64 toolchain support

### PR DESCRIPTION
**Pull request by shahmishal**
Screenshot 2024-07-29 at 10 08 55 PM

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

File: install/windows/index.md [hunk 1]
```diff
@@ -71,7 +73,10 @@ Swift snapshots are prebuilt binaries that are automatically created from the br
     <p class="description">
       Package installers (.exe).
     </p>
+    <ul class="install-instruction">
     <a href="https://download.swift.org/swift-6.0-branch/windows10/{{ windows10_6_0_builds.first.dir }}/{{ windows10_6_0_builds.first.download }}" class="cta-secondary">Download (x86_64)</a>
+    <a href="https://download.swift.org/swift-6.0-branch/windows10-arm64/{{ windows10_arm64_6_0_builds.first.dir }}/{{ windows10_arm64_6_0_builds.first.download }}" class="cta-secondary">Download (arm64)</a>
+    </ul>
```
Comment (alexandersandberg) [inline install/windows/index.md:79]: Children of `<ul>` should be `<li>` for correct HTML markup, but I see this is an issue elsewhere and not only introduced here, so let's leave that fix for another PR.

**Review by alexandersandberg — APPROVED** [review]
